eng/performance/sdk_scenarios.proj

```diff
@@ -17,11 +17,16 @@
     <HelixPreCommands>$(HelixPreCommands);export PYTHONPATH=$HELIX_CORRELATION_PAYLOAD/scripts:$HELIX_CORRELATION_PAYLOAD</HelixPreCommands>
   </PropertyGroup>  
 
+  <PropertyGroup>
+    <FrameworkVersion>$(_Framework.Substring($([MSBuild]::Subtract($(_Framework.Length), 3))))</FrameworkVersion>
+  </PropertyGroup>
+
   <ItemGroup>
       <Framework Include="netcoreapp2.1" FrameworkName="%(Identity)"/>
-      <Framework Include="netcoreapp3.0" FrameworkName="%(Identity)" Condition="'$(_Framework.Substring(10))' &gt; '2.1'"/>
-      <Framework Include="netcoreapp3.1" FrameworkName="%(Identity)" Condition="'$(_Framework.Substring(10))' &gt; '3.0'"/>
-      <Framework Include="netcoreapp5.0" FrameworkName="%(Identity)" Condition="'$(_Framework.Substring(10))' &gt; '3.1'"/>
+      <Framework Include="netcoreapp3.0" FrameworkName="%(Identity)" Condition="'$(FrameworkVersion)' &gt; '2.1'"/>
+      <Framework Include="netcoreapp3.1" FrameworkName="%(Identity)" Condition="'$(FrameworkVersion)' &gt; '3.0'"/>
+      <Framework Include="netcoreapp5.0" FrameworkName="%(Identity)" Condition="'$(FrameworkVersion)' &gt; '3.1'"/>
+      <!-- <Framework Include="net6.0" FrameworkName="%(Identity)" Condition="'$(FrameworkVersion)' &gt; '5.0'"/> -->
```

Comment:
Why is this commented out?

Reply:
because I thought better of it. I didn't want to bite off enabling the SDK tests for 6.0 in this PR, though it needs to happen soon. I will put an item on our taskboard.

Reply:
Removed it in the latest iteration.

Reply:
OK